### PR TITLE
metadata: Add support for Simplify3D v5

### LINKED ISSUE #571: Simplify3D V5 Support
### Is your feature request related to a problem? Please describe

Metadata isnt read or just not showed by mainsail 

### Describe the solution you'd like

Add Metadata support

### Describe alternatives you've considered

_No response_

### Additional information

here is an example gcode: 
[Support_test.zip](https://github.com/Arksine/moonraker/files/10255003/Support_test.zip)

If you need more infos just ask :)

### PR DESCRIPTION
**Pull request by kleinweby**
This PR adds changed to support the new/changed Metadata for Simplify3D Version five.

For most part this are simple changes where only the "property name" changed slightly.

A bot more involved is the new temperature format, as this is a block of properties that which are grouped together and must be parsed as group.

Closes #571.

### CODE REVIEW

**Comment by Arksine** [comment]
Could you attach a sample gcode file sliced with Simplify3D V5 so I could take a look at it?  Thanks.

File: moonraker/components/file_manager/metadata.py [hunk 1]
```diff
@@ -68,12 +69,15 @@ def _regex_find_ints(pattern: str, data: str) -> List[int]:
             pass
     return []
 
-def _regex_find_first(pattern: str, data: str) -> Optional[float]:
+def _regex_find_first(pattern: str,
+                      data: str,
+                      group: int = 1
+                      ) -> Optional[float]:
```
Comment (Arksine) [inline moonraker/components/file_manager/metadata.py:75]: I don't think the addition of a `group` here is necessary, after all this helper is intended to find the first captured group and convert it to a float.  In places where you need to group an expression you should be able use non-capturing groups.  For example:
```
(?:Plastic\sweight|Material\sWeight)
```

Comment (kleinweby) [reply moonraker/components/file_manager/metadata.py]: You're right. I've completely forgot that non-capturing groups exist. Thanks, I've changed the code to use them.

**Comment by kleinweby** [comment]
Here is a sample gcode from Simplify3D v5:

[topRegister.gcode.zip](https://github.com/Arksine/moonraker/files/10787379/topRegister.gcode.zip)

There is also one in #571 which works as well with these changes.

**Comment by Arksine** [comment]
Thanks.  I went ahead and merged this, however I may look to simplify the temperature search code.  It appears to me that the following block is of data is consistent between the two files:
```
;   temperatureController,Primary Extruder
;   enableTemperatureController,1
;   temperatureNumber,0
;   temperatureType,extruder
;   stabilizeAtStartup,1
;   temperatureSetpoints,1|210
;   useIdleCooldown,0
;   cooldownTemperature,150
;   cooldownRequiredIdleTime,180
;   cooldownReheatTime,60
```

Presuming the order doesn't change, we can use a single search with the `re.MULTILINE | re.DOTALL` flags set to find the set point.

**Pull request merged**